crates/services/txpool_v2/src/storage/graph.rs

```diff
@@ -654,6 +671,16 @@ impl Storage for GraphStorage {
                                 }
                             }
                             Ok(None) => {
+                                if saved_outputs.contains(&SavedOutput::Coin(
```

Comment:
Can we put saved output into the variable please?=D It is hard to read=)

Reply:
Done.